src/device-registry/routes/v2/airqlouds.js

```diff
@@ -6,10 +6,15 @@ const constants = require("@config/constants");
 const mongoose = require("mongoose");
 const ObjectId = mongoose.Types.ObjectId;
 const createAirQloudUtil = require("@utils/create-location");
+const AdminLevelSchema = require("@models/AdminLevel");
```

Comment:
@wabinyai / @BenjaminSsempala ....we had forgotten to import the required Schema so the system was just in vibes. Error handling helped us identify the issue.